src/components/KRadio/KRadio.vue

```diff
@@ -101,34 +126,39 @@ const props = defineProps({
     type: [String, Number, Boolean, Object],
     required: true,
   },
+  hasError: {
+    type: Boolean,
+    default: false,
+  },
+  isCard: {
+    type: Boolean,
+    default: false,
+  },
```

Comment:
Since these are new props, can you go ahead and remove the prefixes in this PR?

Reply:
Done